.github/workflows/ci.yml

```
@@ -19,6 +19,19 @@ jobs:
       with:
         dotnet-version: 8.0.x
 
+    - uses: pnpm/action-setup@v4
+      name: Install pnpm
+      with:
+        version: 10
+        run_install: false
+        working-directory: src/ui
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)